schemas/device/MRICoil.schema.tpl.json

```diff
@@ -0,0 +1,53 @@
+{
+  "_extends": "/core/schemas/products/device.schema.tpl.json",
+  "_type": "https://openminds.ebrains.eu/neuroimaging/MRICoil",
+  "properties": {
+    "activeElements": {
+      "_instruction": "If applicable, list the identifiers of the active coil elements or channels.",
+      "type": "string"
+    },
+    "coilGeometry": {
+      "_instruction": "Add the geometry of the coil (e.g., birdcage, loop, Helmholtz).",
+      "_linkedTypes": [
+        "controlledTerms:MRIRadiofrequencyCoilGeometry"
+      ]
+    },
+    "dimension": {
+      "_instruction": "Add the dimention of this coil",
+      "_linkedCategories": [
+        "mathematicalShapes"
+      ]
+    },
+    "elementCount": {
+      "_instruction": "Total number of coil elements.",
+      "minimum": 1,
+      "type": "integer"
+    },
+    "mountingType": {
+      "_instruction": "Add the coil mounting type (e.g., BuiltIn, External, Interventional, Wearable).",
+      "_linkedTypes": [
+        "controlledTerms:MRIRadiofrequencyCoilMountingType"
+      ]
+    },
+    "parallelImagingCapable": {
+      "_instruction": "Indicate whether the coil supports parallel imaging.",
+      "type": "boolean"
+    },
+    "radiofrequencyCoilType": {
+      "_instruction": "Add the coil type (e.g., volume, surface, phased array).",
+      "_linkedTypes": [
+        "controlledTerms:MRIRadiofrequencyCoilType"
+      ]
+    },
+    "targetAnatomyCoverage": {
+      "_instruction": "Add the anatomical region(s) the coil is designed to cover.",
+      "_linkedTypes": [
+        "controlledTerms:AnatomicalRegion"
+      ]
+    }
+  },
+  "required": [
+    "radiofrequencyCoilType",
+    "elementCount"
+  ]
```

Comment:
required properties are usually listed above the properties (i.e., not in alphabetical order as in the final schemas)